src/EA.Weee.Web/Areas/Admin/Controllers/TransferEvidenceReportsController.cs

```diff
@@ -0,0 +1,94 @@
+﻿namespace EA.Weee.Web.Areas.Admin.Controllers
+{
+    using Api.Client;
+    using Services;
+    using System;
+    using System.Linq;
+    using System.Text;
+    using System.Threading.Tasks;
+    using System.Web.Mvc;
+    using Infrastructure;
+    using ViewModels.EvidenceReports;
```

Comment:
create a unit tests for this new class. Use the existing evidence notes tests as a base.